dora/core/common/src/main/java/alluxio/conf/RemovedKey.java

```diff
@@ -100,14 +100,10 @@ private static String replacedSince(String version, String newProperty) {
       put("alluxio.user.block.worker.client.threads", removedSince(V2_0_0));
       put("alluxio.user.failed.space.request.limits", removedSince(V2_0_0));
       put("alluxio.user.file.cache.partially.read.block", removedSince(V2_0_0));
-      put("alluxio.user.file.copyfromlocal.write.location.policy.class", replacedSince(V2_0_0,
```

Comment:
i had to remove from here because there's no `removedSince(v3_0_0)`. I simply don't want to add that functionality in this code-removing PR.
Tracked in https://github.com/Alluxio/alluxio/issues/17852